Doc update (#142)
ltelab/disdrodb: docs/source/data.rst

=========================
Data
=========================

> Access to Disdometer measurement datasets is available through a separate Github repository : `LTE-disdrodb-data <https://github.com/EPFL-ENAC/LTE-disdrodb-data>`__.

Reviewer: ```suggestion
Access to the disdrometer measurement datasets is available through a separate Github repository : `LTE-disdrodb-data <https://github.com/EPFL-ENAC/LTE-disdrodb-data>`__.
```

Reviewer: I gave you adming access. Move the repo to the ltelab, rename it disdrodb-data and then modify this accordingly?

Reviewer 2: @regislon ping to do this :)

> This Github repository includes the following information:

Reviewer: ```suggestion
This GitHub repository includes the following information:
```

> - Station measurements in the form of a JSON file with links to the downloadable data (not included in the repository due to Github's file size limitation).

Reviewer: ```suggestion
- Stations measurements in the form of a JSON file with links to the downloadable data (not included in the repository due to GitHub's file size limitation).
```

> - Station metadatas in a yaml files.

Reviewer: ```suggestion
- Stations metadata in YAML files.
```

> - Station issues in a CSV files.

Reviewer: ```suggestion
- Stations issues in YAML files.
```

Reviewer: This has not been yet implemented right?

> - Python code to download the dataset using the URLs provided in the JSON file.

Reviewer: ```suggestion
- Python code to download the datasets using the URLs provided in the JSON files.
```
